code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -887,6 +884,10 @@ GLOBAL_LIST_EMPTY(PDAs)
 		if(add_photo=="Yes")
 			var/datum/picture/Pic = aicamera.selectpicture(aicamera)
 			src.aiPDA.photo = Pic.fields["img"]
+
+	if(incapacitated())

```

Comment:
Any reason you moved it from 863?

Reply:
The initial check is already handled by the HUD object but we need to check again in case the AI was depowered while the input window is open.